website_container_fluid/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
As this icon is the sample one without modifications, you don't need to provide it.